python/ray/serve/controller.py

```diff
@@ -155,6 +155,24 @@ class ActorStateReconciler:
     backends_to_remove: List[BackendTag] = field(default_factory=list)
     endpoints_to_remove: List[EndpointTag] = field(default_factory=list)
 
+    # NOTE(ilr): These are not checkpointed, but will be recreated by
+    # `_enqueue_kill_actor`.
```

Comment:
Is this comment accurate? I don't see a method called `_enqueue_kill_actor`?

Reply:
I meant `_enqueue_pending_scale_changes_loop`